apps/easypid/.env

```diff
@@ -0,0 +1,2 @@
+EXPO_PUBLIC_WALLET_SERVICE_PROVIDER_AUTH_TOKEN=0dd5e0ac-f6a5-42ce-b945-09735c3a8f17
```

Comment:
Auth token doesn't really work for mobile apps as you can extract it from the build.

Or is this temporary until we can do app integrity?

Reply:
This was very temporary. Also, I should not push the .env file haha